dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/session/impl/SessionAffinityManagerImpl.java

```diff
@@ -505,29 +507,34 @@ public Cookie run() {
                             cookie.setSecure(true);
                         }
 
-                        WebContainerRequestState requestState = WebContainerRequestState.getInstance(true);
                         String sameSiteCookieValue = sessionSameSiteCookie.getSameSiteCookieValue();
                         requestState.setCookieAttributes(cookie.getName(), "SameSite=" + sameSiteCookieValue);
-
-                        if (ProductInfo.getBetaEdition()) {
-                            // not null means a user defined config was set
-                            if(_smc.getSessionCookiePartitioned() != null && sessionSameSiteCookie.equals(SameSiteCookie.NONE)) {
-                                if(_smc.getSessionCookiePartitioned()) {
+                    } 
+
+                    if (ProductInfo.getBetaEdition()) {
+                        // not null means a user defined config was set
+                        if (_smc.getSessionCookiePartitioned() != null) {
+                            if (_smc.getSessionCookiePartitioned()) {
+                                // None is okay and disabled is okay -- http config may partition all cookies
+                                if (sessionSameSiteCookie != SameSiteCookie.LAX && sessionSameSiteCookie != SameSiteCookie.STRICT) {
```

Comment:
Can you explain "disabled is okay"?

Reply:
We don't want to set Partitioned on Lax or Strict (since it does nothing).

None / Disabled is okay 

None --  Partitioned matches with none, so go ahead an set it.

Disabled - http config may set samesite=none all cookies.

Reply:
I added this comment so that this check was intentional

Reply:
Maybe I should have said `http config may set samesite=none all cookies`